release_notes/ocp-4-0-release-notes.adoc

```diff
@@ -0,0 +1,795 @@
+[id='ocp-4-1-release-notes']
+= {product-title} 4.1 release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid
+cloud application platform for deploying both new and existing applications on
+secure, scalable resources with minimal configuration and management overhead.
+{product-title} supports a wide selection of programming languages and
+frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Kubernetes, {product-title}
+provides a more secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while delivering integrated application runtimes
+and libraries. {product-title} enables organizations to meet security, privacy,
+compliance, and governance requirements.
+
+[[ocp-4-1-about-this-release]]
+== About this release
+
+Red Hat {product-title} 4.1
+(link:https://access.redhat.com/errata/RHBA-2019:XXXX[RHBA-2019:XXXX]) is now
+available. This release is based on OKD 4.1, and it uses Kubernetes 1.13. New
+features, changes, and known issues that pertain to {product-title} 4.1 are
+included in this topic.
+
+{product-title} 4.1 is supported on Red Hat Enterprise Linux 8 and later, as
+well as Red Hat CoreOS 8.
+
+[[ocp-41-deprecated-features]]
+=== Deprecated features
+
+Large changes to the underlying architecture and installation process are
+applied in {product-title} 4.1, and many features from {product-title} 3.x are
+now deprecated.
+
+.Features Deprecated in Version 4.1
+[cols="2",options="header"]
+|====
+|Feature |Justification
+
+|Hawkular
+|Replaced by cluster monitoring.
+
+|Cassandra
+|Replaced by cluster monitoring.
+
+|Heapster
+|Replaced by Prometheus adapter.
+
+|Atomic Host
+|Replaced by Red Hat CoreOS.
+
+|System containers
+|Replaced by Red Hat CoreOS.
+
+|projectatomic/docker-1.13 additional search registries
+|CRI-O is the default container runtime for 4.x on RHCOS and Red Hat Enterprise Linux.
+
+|`oc adm diagnostics`
+|Operator-based diagnostics.
+
+|`oc adm registry`
+|Replaced by the registry operator.
+
+|Custom Docker Build Strategy on Builder Pods
+|If you want to continue using custom builds, you must replace your Docker
```

Comment:
@bparees @bmcelvee debating on replacing the _must_ qualifier with _should_.

Technically you can continue to use docker, but the customer must:
1. BYOH using RHEL7 nodes
2. Install Docker from the extras - this will not be supported in later versions of RHEL7
3. Continue exposing the docker socket in the `BuildConfig` spec

This is the first deprecation notice. I don't think we will ever get rid of that field in the API, but in a later OpenShift release it will stop being functional.

Reply:
yeah, agree it's a should not a must.